docs/Operators.md

```diff
@@ -2687,6 +2687,30 @@ expect(node, inputs=[x, W], outputs=[y],
 </details>
 
 
+<details>
+<summary>convtranspose_dilations</summary>
+
+```python
+x = np.array([[[[3., 8., 1.],  # (1, 1, 3, 3)
+                [9., 5., 7.],
+                [3., 2., 6.]]]]).astype(np.float32)
+W = np.array([[[[7., 2.],  # (1, 1, 2, 2)
+                [1., 9.]]]]).astype(np.float32)
+
+node = onnx.helper.make_node("ConvTranspose", ["X", "W"], ["Y"], dilations=[2, 2])
+
+y = np.array([[[[21., 56., 13., 16., 2.],  # [1, 1, 5, 5]
```

Comment:
I checked, the results I got was:

tensor([[[[21.1887, 56.1887, 13.1887, 16.1887,  2.1887],
          [63.1887, 35.1887, 67.1887, 10.1887, 14.1887],
          [24.1887, 22.1887, 76.1887, 76.1887, 21.1887],
          [ 9.1887,  5.1887, 88.1887, 45.1887, 63.1887],
          [ 3.1887,  2.1887, 33.1887, 18.1887, 54.1887]]]]

Did I miss something here?

Reply:
Mind to share your script here?

Reply:
Never mind, I forgot to set bias. Apparently, that noise is from bias.

The result look good to me now.